lib/to_openstudio/model.rb

```diff
@@ -89,6 +89,7 @@ def to_openstudio_model(openstudio_model=nil, log_report=true)
                             OpenStudio::Model::Model.new
                           end
 
+      @openstudio_model.getYearDescription.setCalendarYear(2020)
```

Comment:
We should probably set this to 2017 since the Ladybug Tools default is to have Jan 1 be on a Sunday.

Reply:
Or it should be 2016 if the EPW is for a leap year. Hopefully, this will just correct itself when the EPW gets assigned.